apps/laboratory/next.config.js

```diff
@@ -1,11 +1,34 @@
 /** @type {import('next').NextConfig} */
+const cspHeader = `
+  default-src 'self';
+  script-src 'self' 'unsafe-inline';
+  style-src 'self' 'unsafe-inline' https://fonts.googleapis.com;
+  img-src 'self' data: blob: https://walletconnect.org https://walletconnect.com https://secure.walletconnect.com https://secure.walletconnect.org https://avatars.githubusercontent.com https://tokens-data.1inch.io https://tokens.1inch.io https://ipfs.io;
```

Comment:
@chris13524 is `https://avatars.githubusercontent.com/` being used by W3M?

Reply:
Yes it is used for the examples/laboratory and [the Solana package uses it too (a bug)](https://walletconnect.slack.com/archives/C03RVH94K5K/p1718060201496369).

For the examples/laboratory perhaps we can self-host though in order to remove this

Reply:
Oh wait the examples/laboratory doesn't use it because it's actually react-wallet that uses it. So not a blocker here

Reply:
Needed it for the W3M actually (for the sign/connect flow). I added it to this repo.